imap/lmtp_sieve.c

```diff
@@ -785,7 +785,13 @@ static int send_forward(sieve_redirect_context_t *rc,
 #endif
     }
     else {
-        smtp_envelope_add_rcpt(&sm_env, rc->addr);
+        struct address *addr = NULL;
+        char *rcpt = NULL;
+
+        parseaddr_list(rc->addr, &addr);
+        rcpt = address_get_all(addr, 1);
+        smtp_envelope_add_rcpt(&sm_env, rcpt);
+        free(rcpt);
```

Comment:
Think you want a `parseaddr_free(addr)` in here too, that `struct address` type looks like a dynamically-allocated linked list

Reply:
Yep.  Good catch!.  FIxed